lib/BridgedRoom.js

```diff
@@ -3,6 +3,7 @@
 var Promise = require("bluebird");
 
 var retry = require("./retry");
+const log = require("./Logging.js").Get("Provisioning");
```

Comment:
Probably best to be consistent with the context. This should be `"BridgedRoom"`, no?